### PR TITLE
front : disable date and arrival time stdcm

### LINKED ISSUE #5636: Arrival time in STDCM should be removed
### What happened?

The radio button is disabled here but not the date and time inputs :
image

### What did you expect to happen?

This feature will be implemented later so this component should not be displayed anymore during that time.

More over, the radio button of the previous component should be removed as there will be only one possibility.

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to STDCM

### What operating system, browser and environment are you using?

- **Env**: statging

### OSRD version (top right corner `Account` button > `Informations`)

aa9ff62c

### PR DESCRIPTION
**Pull request by RomainValls**
closes #5636

### CODE REVIEW

File: front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Destination.tsx
Comment (SharglutDev) [inline front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Destination.tsx]: You left a comment there

File: front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Destination.tsx
Comment (SharglutDev) [inline front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Destination.tsx]: same here

**Review by flomonster — COMMENTED** [review]
Tested. You can't visually see that the elements are deactivated. This issue might not be part of this PR.

**Review by SharglutDev — CHANGES_REQUESTED** [review]
Thank you for the PR. Seems to work as expected but I would like to wait for the PO's / UX approval to be sure that's the behavior we want.

PS : don't forget to remove your comments

**Review by SharglutDev — CHANGES_REQUESTED** [review]
Don't forget to remove the `{radioButton}` const and its call in the tsx in `Origin.tsx` component 

And maybe you can do only 1 commit for this PR

**Review by SharglutDev — APPROVED** [review]
Lgtm, tested and everything looks as expected, good job !

Thank you for squashing your old commits.

File: front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Origin.tsx
Comment (clarani) [inline front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Origin.tsx]: Can you also remove the isByDestination here please ? Thanks ! :)

**Review by clarani — CHANGES_REQUESTED** [review]
Nice job ! Can I ask you to remove some dead code as we are removing these radio buttons ?
- remove updateStdcmMode function (in reducer and in slice) since it will not be used anymore
- remove getStdcmMode selector
- remove isByDestination in Origin
- in formatStdcmCong, remove isByOrigin and isByDestination booleans. You can also remove the test `if (!osrdconf.destinationTime && isByDestination) {...}` line 48 but you need to keep the test line 30 (only remove isByOrigin test)

**Review by clarani — APPROVED** [review]
LGTM, it works perfectly !! :white_check_mark: 
Thanks for this PR !

**Pull request merged**
